mtags/src/main/scala/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -175,14 +177,13 @@ class HoverProvider(val compiler: MetalsGlobal, params: OffsetParams) {
     else if (symbol.hasPackageFlag || symbol.hasModuleFlag) {
       Some(
         new Hover(
-          List(
-            JEither.forRight[String, MarkedString](
-              new MarkedString(
-                "scala",
-                s"${symbol.javaClassSymbol.keyString} ${symbol.fullName}"
-              )
-            )
-          ).asJava
+          new MarkupContent(
+            MarkupKind.MARKDOWN,
```

Comment:
I misunderstood this at first and put `scala`, but it looks like this should be `plainText` or `markdown` according to the spec. Then , you put ````scala` in the actual value. They give an example in the LSP docs.

https://microsoft.github.io/language-server-protocol/specifications/specification-current/#markupContent